### PR TITLE
[IN-368] api.mesh

### PR DESCRIPTION
**Pull request by sureshponraj90**

### CODE REVIEW

File: lib/guestDrawer.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	const counterState = value(1);
+	const calculationsState = array([]);
+
+	const state = api.observ.state({
+		input: value(undefined),
+		output: value(undefined)
+	});
+
+	// Load our WebAssembly API
+	const wasmApi = WebAssembly.instantiateStreaming(fetch('https://plugins.coviu.com/examples/wasm/test.wasm')).then(({ instance, module }) => {
```
Comment (jerry-coviu) [inline lib/guestDrawer.js:22]: what's this for?

**Review by jerry-coviu — COMMENTED** [review]
Can we make this mesh into another plugin? So we would have 
- simple addon with resource for most of addon developer
- addon with resource with mesh

File: lib/guestDrawer.js (hunk 1 above)
Comment (sureshponraj90) [reply lib/guestDrawer.js:22]: This is a Webassembly to return some random numbers. This can be done via webhook in the next card

**Comment by sureshponraj90** [comment]
@jerry-coviu Having 2 demo plugins might create lots of duplicate code and also might be confusing for external devs.

File: lib/guestDrawer.js (hunk 1 above)
Comment (jerry-coviu) [reply lib/guestDrawer.js:22]: undersood, thanks

**Comment by jayenashar** [comment]
> @jerry-coviu Having 2 demo plugins might create lots of duplicate code and also might be confusing for external devs.

i agree.  one plugin with different functions, clearly separated in folders and UI elements would be less confusing, and provide a template for devs to cut/copy and paste.

**Comment by jerry-coviu** [comment]
LGTM, nicely done. We improve more later on

**Review by jerry-coviu — APPROVED** [review]

**Pull request merged**